hooks/module_config_validator.py

```diff
@@ -86,16 +87,28 @@ def r(ctx: hook.Context):
                 .get("filterResult", {})
                 .get("cidrs", [])
             ]
-            self.check_overlaps_cidrs(cidrs)
+
+            try:
+                self.check_overlaps_cidrs(cidrs)
+            except ValueError as e:
+                print(f"ERROR: {e}")
+
+                self.set_value(self.path, ctx.values, False)
```

Comment:
Will it ends up here if there are no "cidrs" in ModuleConfig?
It seems there should be another check to set isReady=False if "cidrs" are empty.